services/hanke-service/src/main/resources/db/changelog/changesets/050-make-nimi-mandatory-in-hanke.yml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 050-make-nimi-mandatory-in-hanke
+      author: Topias Heinonen
+      changes:
+        - addNotNullConstraint:
+            tableName: hanke
+            columnName: nimi
+            # There shouldn't be unnamed hanke, but let's add a default name
+            # just in case, instead of failing to start.
+            defaultNullValue: "Unnamed"
```

Comment:
Unnamed would show up in the ui also. As stated, there should not be unnamed instances, but could this be something that works in Finnish, Swedish and English? Would an empty string work if there is no other suitable name?

Reply:
I checked every environment and there are no hanke with a null name in any of them. I removed the default value.